Patches/Core/ThingDefs_Misc/Weapons_Melee.xml

```diff
@@ -581,7 +581,7 @@
 					<capacities>
 						<li>Stab</li>
 					</capacities>
-					<power>18</power>
```

Comment:
It makes no sense that stabs from gladius or ikwa, both one handed weapons, does more damage on stab than the longsword, a two handed weapon. The spear does fine with low damage per hit since it has a very low melee cooldown. Longswords gets rubbish damage and a long cool down.
Maybe the lower both the damage and cooldown a little but its just bizzare how little damage a long sword stab does for how slow it is.

Reply:
Also, I would like to add that the warhammer is currently doing too much damage per hit, causing many one hit deaths which largely ignores any medieval tier armour.
I might make a separate PR taking another look at the different melee weapons. If this is the not the appropriate PR for this, feel free to merge this patch without any changes to the long sword.

Reply:
It's not the correct PR for it, and I'll need you to revert the commit/file on the branch you're PRing from, as I can't selectively merge the file changes.